### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Yonashenok**
## Summary 
[@Yonashenok](https://github.com/Yonashenok) I  have completed the implementation of [Vet clinic database: create animals table](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) on the Vet clinic database according to project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md). 

The following tasks have been implemented: 
- [x] Setup [PostgreSQL](https://www.postgresql.org/).
- [x] Create a database named `vet_clinic`.
- [x] Create a table named animals with (`id`, `name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) 
- [x] Insert all the give data
- [x] Write queries according to requirement 
    * Find all animals whose name ends in `"mon"`.
 image
   * List the name of all animals born between 2016 and 2019. 
image
  * List the name of all animals that are neutered and have less than 3 escape attempts.
image
  * List the date of birth of all animals named either "Agumon" or "Pikachu".
image
  * List name and escape attempts of animals that weigh more than 10.5kg
image
  * Find all animals that are neutered.
image
  * Find all animals not named Gabumon. 
image
  * Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
image

 

In addition, I have fulfilled the following default requirements:

* Used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
* Documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

I will be  looking forward to your feedback.
Thank you!

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
```
Comment (thecodechaser) [inline queries.sql]: - [ ] This query is not part of the requirements, Please make sure to remove this.

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
 CREATE TABLE animals (
-    name varchar(100)
+    id  int,
+    name varchar(80),
+    data_of_birth   date,
+    escape_attempts  int,
+    neutered  bool,
+    weight_kg  decimal
 );
+
```
Comment (thecodechaser) [inline schema.sql]: - [ ] [OPTIONAL] I would suggest marking the id column as the primary key.
```ruby
PRIMARY KEY (id)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Yonashenok👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with the animals' table. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Yonashenok,

✔️ Creation of animals table

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**